src/inputs/Transforms.js

```diff
@@ -148,21 +151,44 @@ class Shape extends Component {
     }
 
     var options_render = this.props.options.map( (option) => {
-      return <FormGroup controlId="options-step" key={option.title}>
-               <Col componentClass={ControlLabel} sm={4}>
-                 {option.title}
-               </Col>
-               <Col sm={8}>
-                 <FormControl
-                   type={option.type ? option.type : "number"}
-                   step={option.step ? option.step : 1}
-                   value={option.value()}
-                   onChange={(event) => {
-                     option.onChange(event)
-                   }}
-                   onKeyDown={disableEnter}/>
-               </Col>
-             </FormGroup>
+      if (option.type && option.type === "dropdown") {
+        return <FormGroup controlId="options-step" key={option.title}>
+                 <Col componentClass={ControlLabel} sm={4}>
+                   {option.title}
+                 </Col>
+                 <Col sm={8}>
+                   <ButtonGroup justified>
```

Comment:
Personal preference, but I would remove the ButtonGroup here. The non-justified dropdown button looks better, less heavy, and left-aligned.